pkg/controller/storageprofile-controller.go

```diff
@@ -132,6 +132,17 @@ func (r *StorageProfileReconciler) reconcilePropertySets(sc *storagev1.StorageCl
 	return claimPropertySets
 }
 
+func (r *StorageProfileReconciler) reconcileCloneStrategy(sc *storagev1.StorageClass, clonestrategy *cdiv1.CDICloneStrategy) *cdiv1.CDICloneStrategy {
+
+	if clonestrategy == nil {
+		if sc.Annotations["cdi.kubevirt.io/clone-strategy"] == "host-assisted" {
```

Comment:
This looks very good, but as we are adding the clone-strategy annotation I would like to see all 3 values for cdiv1.CDICloneStrategy covered here. 
So:
```
	// CloneStrategyHostAssisted specifies slower, host-assisted copy
	CloneStrategyHostAssisted = "copy"

	// CloneStrategySnapshot specifies snapshot-based copying
	CloneStrategySnapshot = "snapshot"

	// CloneStrategyCsiClone specifies csi volume clone based cloning
	CloneStrategyCsiClone = "csi-clone"
```

Also please be consistent, with configuration options so use `copy, snapshot and csi-clone`.

It also should be really easy to extend test to (use `DescribeTable` with multiple Entries like here: pkg/apiserver/webhooks/cdi-validate_test.go:86)

Reply:
@brybacki  i have addressed the changes.
And you mean to follow the DescribeTable structure in implementing the testcases in storageprofile-controller_test.go? Right now i have used It() -> expect model